src/main/java/dino/command/AddTaskCommand.java

```diff
@@ -0,0 +1,123 @@
+package dino.command;
+
+import java.time.format.DateTimeParseException;
+
+import dino.util.Parser;
+import dino.util.Storage;
+import dino.exception.*;
+import dino.task.Deadline;
+import dino.task.Event;
+import dino.task.Task;
+import dino.task.TaskList;
+import dino.task.ToDo;
+import java.time.LocalDate;
+
+
+/**
+ * Represents the command for which the user wants to add a task to the task list
+ * The task can only be one of the task of type ToDo, Deadline, or Event
+ */
+public class AddTaskCommand extends Command {
+
+    private final String taskString;
+    private final Parser.CMDTYPE taskType;
+
+    /**
+     * Constructs a AskTaskCommand object
+     *
+     * @param taskString the user input command
+     * @param taskType the type of the task, which can only be one of the type ToDo, Deadline, or Event
+     */
+    public AddTaskCommand(String taskString, Parser.CMDTYPE taskType) {
```

Comment:
hm, this is interesting. May I know why you decided to group the adding of tasks as one command, instead of separating them such as "AddListCommand", "AddDeadlineCommand"? ... I think as the code-base gets bigger, and if you do choose to implement more different types of tasks, this class and in particular the execute and getTaskDescription methods can get pretty messy.